new-pipeline/src/dashboards.js

```diff
@@ -1081,6 +1081,23 @@ function getDescriptionLink(metric, channel, description) {
       class: "btn btn-outline-primary fa fa-info-circle",
     }).text(" More details"));
   }
-
   return link;
 }
+
+function getUseCounterLink(metric, channel, description) {
+  var metricUrl = buildDictionaryURL(metric, channel, description);
+  if (metric.startsWith("USE_COUNTER") !== true) {
+    $(".use-counter-link").text(""); // Clear use counter link;
```

Comment:
I would suggest to leave dealing with DOM elements to dist.js & evo.js.
So we could do something like:
- default the use counter link to hidden and add it's label to the HTML.
- return `null` from `getUseCounterLink()`, unless its a `USE_COUNTER*`, in which case we return something (the URL to link to or an `<a>` element).
- in dist.js & evo.js, if `getUseCounterLink()` returns `null`, hide/clear the element. Otherwise update & show the link.

Reply:
Is there an ideal placement for the if/else statements in dist.js and evo.js? Should I keep them within the `displayHistograms` function and the main `telemetry.init` function, respectively?

Reply:
Keeping them close to `getDescriptionLink()` should work.
That function covers somewhat similar needs.

Reply:
Makes sense! :)